### PR TITLE
Security code fix after testing.

### PR DESCRIPTION
**Pull request by badhrinathpa**

### CODE REVIEW

File: src/mme-app/mmeStates/s1HoWfModifyBearerResponse.cpp [hunk 1]
```diff
@@ -63,4 +64,10 @@ void S1HoWfModifyBearerResponse::initialize()
                 actionTable.setNextState(S1HoWfTauCheck::Instance());
                 eventToActionsMap.insert(pair<uint16_t, ActionTable>(MB_RESP_FROM_SGW, actionTable));
         }
+        {
+                ActionTable actionTable;
```
Comment (thakurajayL) [inline src/mme-app/mmeStates/s1HoWfModifyBearerResponse.cpp:68]: this is auto generated code. I dont see corresponding change in the json files under the script folder.

File: src/s1ap/handlers/s1ap_tau_response_handler.c [hunk 2]
```diff
@@ -77,7 +77,7 @@ get_tau_rsp_protoie_value(struct proto_IE *value, struct tauResp_Q_msg *g_tauRes
 #endif
 
 	free(value->data[2].val.nas.elements);
-	free(value->data);
+	//free(value->data);
```
Comment (thakurajayL) [inline src/s1ap/handlers/s1ap_tau_response_handler.c:80]: I dont see its getting released anywhere else. Can you please tell me the line # at which its getting freed.

Comment (badhrinathpa) [reply src/s1ap/handlers/s1ap_tau_response_handler.c:80]: line 152 and 363. May be this is already updated in the master branch.

File: src/mme-app/mmeStates/s1HoWfModifyBearerResponse.cpp (hunk 1 above)
Comment (badhrinathpa) [reply src/mme-app/mmeStates/s1HoWfModifyBearerResponse.cpp:68]: I merged this to test. Left the json part cause anyways its part of master and tested.

**Pull request merged**